src/ledger/LedgerTestUtils.cpp

```diff
@@ -102,26 +104,34 @@ makeValid(TrustLineEntry& tl)
     clampLow<int64_t>(0, tl.balance);
```

Comment:
clampLow for balance and limit are redundant considering that balance and limit are already converted to absolute value. This PR is an opportunity to fix this

Reply:
fixed